lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -364,7 +364,12 @@ export class UserAgentApplication {
     }
 
     // Validate and filter scopes (the validate function will throw if validation fails)
-    this.validateInputScope(scopes, false);
+    try {
```

Comment:
rethrowing adds unnecessary complexity in the code. We shouldn't use this, and there are multiple locations for this.